doc/man5/config.pod

```diff
@@ -216,6 +221,57 @@ For example:
  # Supply all default algorithms
  default_algorithms = ALL
 
+=head2 Provider Configuration Module
+
+This provider configuration module has the name B<providers>. The
+value of this variable points to a section containing further provider
+configuration information.
+
+The section pointed to by B<providers> is a table of engine names
```

Comment:
Engine names?  If we're trying to move away from engines, it might be better to write _provider names_.

Reply:
Specially since this is about providers! :wink:

Reply:
Fixed